conf/application.conf

```diff
@@ -84,7 +84,7 @@ microservice {
         port = 9570
         protocol = http
         redirect-base = "http://localhost:10018"
-        max-file-size-in-mb = 3
+        max-file-size-in-mb = 20
```

Comment:
this needs changed back to 3mb before being merged. If it needs tested in staging it should be overridden in app-config-staging. Other than that LGTM

Reply:
Done